charts/value-dev.yml

```diff
@@ -1,6 +1,4 @@
 zenko-zookeeper:
-  Image: localhost:5000/zookeeper
```

Comment:
How does @Zempashi feel about this? I removed it also at first, he thought it should stay. What's the final call?

Reply:
What is the benefit of keeping a link to the local registry since we have an official image on dockerhub. It makes sense to also support having all zenko images into a local registry for some specific deployments.

Reply:
@Zempashi's rationale was that a developer may wish to deploy using some (locally modified) version of the zenko-zookeeper image.

Reply:
There is two kinds of dev here: dev on charts (and here it make sense to keep the images from the repositories) and dev on image (and it's a pain on kubernetes without something like above). We wan create two separate files, or now that an image is release: just manually override this when working on docker image. 
The rational for me to keep this as the default behaviour: if there's change an a docker file and the helm (to reflect change in the docker image). The helm install will fail on that branch (until the branch is 'released'). If it's acceptable go ahead to remove this.